public/events/index.html

```diff
@@ -229,8 +229,11 @@ <h4 class="panel-title">
             </div>
           </div>
           <div id="contacts" class="tab-pane fade in active">
-            <div class="col-md-6" >
+            <div class="col-sm-12" >
```

Comment:
Making this col-sm-12 breaks the layout (pushes the map to the bottom, and breaks the js code around L300 that looks for this div when switching between tabs). Will revert in a subsequent commit.

Reply:
Actually, @mehrdadgit @rcdelfin , the col sizes are now totally out of date with the js code. Can you please fix all as I don't have time to fix them all for another week or so due to travel.

Reply:
OK @matthewberryman I had a quick look (after you reverted) do not get which part is failing ? will have a deeper look soon...

Reply:
Fixed under issue #122; note @matthewberryman as only the parent div sizes are dynamically changing (sidepane and mapContainer) the col-md/sm-12 size on this line will be correct as it just means width=100% (of its parent div)